### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Mollinn**
- [DEMO LINK](https://Mollinn.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Mollinn.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

**Comment by Mollinn** [comment]
в сенсі тести не пройдені якщо пройдені?????

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,50 @@
...
+      <div class="form__big-icon icon-form">
+        <img
+          class="form__big-img"
+          src="../src/images/Search.svg"
+          alt="icon search"
+        >
+      </div>
+      <label for="1"></label>
+      <input
+        id="1"
```
Comment (mykhalenych) [inline src/index.html:29]:
```suggestion
        id="bigInput"
```

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,50 @@
...
+      class="form"
+    >
+      <div class="form__big-icon icon-form">
+        <img
+          class="form__big-img"
+          src="../src/images/Search.svg"
+          alt="icon search"
+        >
+      </div>
+      <label for="1"></label>
```
Comment (mykhalenych) [inline src/index.html:27]:
```suggestion
      <label for="bigInput"></label>
```

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,50 @@
...
+      <div class="form__small-icon icon-form">
+        <img
+          class="form__small-img"
+          src="../src/images/Search.svg"
+          alt="icon search"
+        >
+      </div>
+      <label for="2"></label>
+      <input
+        id="2"
```
Comment (mykhalenych) [inline src/index.html:52]:
```suggestion
        id="smallInput"
```

File: src/index.html [hunk 4]
```diff
@@ -11,6 +11,50 @@
...
+      action="/action_page.php"
+    >
+      <div class="form__small-icon icon-form">
+        <img
+          class="form__small-img"
+          src="../src/images/Search.svg"
+          alt="icon search"
+        >
+      </div>
+      <label for="2"></label>
```
Comment (mykhalenych) [inline src/index.html:50]:
```suggestion
      <label for="smallInput"></label>
```

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+.form__big-img {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.big__input {
+  font-family: "Avenir", sans-serif;
```
Comment (mykhalenych) [inline src/style.css:49]: you already added font-family in body selector, delete in all other places

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+.small__input {
+  font-family: "Avenir", sans-serif;
+  width: 100%;
+  height: 42px;
+  color: var(--text-color);
+  padding-left: 33px;
+  border-radius: 4px;
+  border: 1px solid var(--border-color);
```
Comment (mykhalenych) [inline src/style.css:69]: don't repeat your self, you duplicate styles for inputs, create one class with common styles and reuse for two inputs

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+
+.form__small-img {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.input-big::placeholder {
```
Comment (mykhalenych) [inline src/style.css:105]: why do you create this class?
you can use big__input

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+  color: inherit;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.input-small::placeholder {
```
Comment (mykhalenych) [inline src/style.css:111]: the same with small

**Review by mykhalenych — CHANGES_REQUESTED** [review]
Nice work, fix small suggestions

**Pull request closed without merging**